src/shared/hibernate-util.c

```diff
@@ -28,446 +28,425 @@
 
 #define HIBERNATION_SWAP_THRESHOLD 0.98
 
-SwapEntry* swap_entry_free(SwapEntry *se) {
-        if (!se)
-                return NULL;
-
-        free(se->path);
+void hibernate_device_done(HibernateDevice *hibernate_device) {
+        assert(hibernate_device);
 
-        return mfree(se);
+        free(hibernate_device->path);
 }
 
-HibernateLocation* hibernate_location_free(HibernateLocation *hl) {
-        if (!hl)
-                return NULL;
+int read_fiemap(int fd, struct fiemap **ret) {
```

Comment:
read_filemap?

Reply:
Hmm, I didn't touch this part. And I think it makes sense to say `fiemap` as the struct name.